airflow/mocks/PodmanBind.go

```diff
@@ -0,0 +1,318 @@
+// Code generated by mockery v2.9.4. DO NOT EDIT.
+
+package mocks
+
+import (
+	context "context"
+
+	auth "github.com/containers/common/pkg/auth"
+	containers "github.com/containers/podman/v3/pkg/bindings/containers"
+
+	entities "github.com/containers/podman/v3/pkg/domain/entities"
+
+	handlers "github.com/containers/podman/v3/pkg/api/handlers"
+
+	images "github.com/containers/podman/v3/pkg/bindings/images"
+
+	mock "github.com/stretchr/testify/mock"
+
+	play "github.com/containers/podman/v3/pkg/bindings/play"
+
+	podman "github.com/astronomer/astro-cli/pkg/podman"
+
+	pods "github.com/containers/podman/v3/pkg/bindings/pods"
+
+	types "github.com/containers/image/v5/types"
+)
+
+// PodmanBind is an autogenerated mock type for the PodmanBind type
+type PodmanBind struct {
+	mock.Mock
+}
+
+// Build provides a mock function with given fields: ctx, containerFiles, options
+func (_m *PodmanBind) Build(ctx context.Context, containerFiles []string, options entities.BuildOptions) (*entities.BuildReport, error) {
+	ret := _m.Called(ctx, containerFiles, options)
```

Comment:
not sure if im missing context but it would be cool if we can refactor these names since im not sure what they are representing, ret, r0, rf

Reply:
for this whole file, if possible

Reply:
This is an auto generated file from [mockery](https://github.com/vektra/mockery) to generate mocks for `PodmanBind` interface, it is being used to write unit tests. Usually any files under mocks folder are autogenerated, hence I didn't mention in the PR description. There is no point in editing this file, as it will be overwritten whenever someone will try to regenerate the mocks.

Reply:
sorry didnt see it was a mock